pkg/cloud/google/machineactuator.go

```diff
@@ -689,7 +693,14 @@ func (gce *GCEClient) updateMasterInplace(cluster *clusterv1.Cluster, oldMachine
 	return nil
 }
 
+func stripVersion(version string) (newVersion string) {
+	cutSet := "vV"
+	newVersion = strings.Trim(version, cutSet)
+	return
```

Comment:
This could be a one liner

```suggestion
	return strings.Trim(version, "vV")
```